tools/parity_scale_decoder/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
+# Disable colour in error tracebacks. Useful for Docker-based deployment.
 no-color = ["colored/no-color"]
```

Comment:
Not exactly right. This is not for error tracebacks, it's for all program output (noticeable when running i.e. `cargo run --bin parity_scale_decoder -- decode tools/parity_scale_decoder/samples/trigger.bin`).

But yeah, it is useful for Docker and terminals without color support

Reply:
Ah, my mistake.

Reply:
fixed